app/models/report.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+# == Schema Information
+#
+# Table name: reports
+#
+#  id          :integer          not null, primary key
+#  reporter_id :integer
+#  reportee_id :integer
+#  reasons     :text
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#  comment_id  :integer
+#  user_id     :integer
+#
+
+class Report < ApplicationRecord
+  belongs_to :user
```

Comment:
In that case, you could update the models relationship by specifying the foreign key that points to the users table. In this case, `reporter_id`:
`belongs_to :user, class_name: 'User', foreign_key: 'reporter_id'`